### ISSUE TITLE
ASA 5525-X not supported

### ISSUE THREAD

**Issue by andrewpwade** [ytti/oxidized#66]
Hi,

I'm testing with oxidized v0.2.3. Hardware is Cisco ASA 5525-X. I tried 'asa' and 'ios' models in the configuration. N.B. we may not have have hostnames/DNS set up correctly.

hardware details:

```
FW-XX-001> show version

Cisco Adaptive Security Appliance Software Version 9.0(1) 
Device Manager Version 7.0(2)

Compiled on Fri 26-Oct-12 17:15 PDT by builders
System image file is "disk0:/asa901-smp-k8.bin"
Config file at boot was "startup-config"

FW-XX-001 up 1 year 30 days
failover cluster up 2 years 45 days

Hardware:   ASA5525, 8192 MB RAM, CPU Lynnfield 2394 MHz, 1 CPU (4 cores)
            ASA: 4096 MB RAM, 1 CPU (1 co
```

When I ssh, this is the output:

````
admin@1.1.1.1's password: 
Type help or '?' for a list of available commands.
FW-XX-001>
```

The last line of the ssh input log is "FW-XX-001> ?" and no backup is made.

Also, regarding recent commits:

* c59b4b5c0605f5f09721bc04d48c1894dfc7694d there is no slash in my prompt (I believe you have to enable this via "prompt hostname state priority")
* 34c4559c845bd6ba239bbf1013fcebbd8041e6c3 there's no '@' in my prompt

**Comment by ytti** [comment]
Pull request maybe with new ASA model with better prompt? Pretty please :)

**Comment by sts** [comment]
@andrewpwade the slash comes when you run two ASAs in an eg. active/standby configuration and shows the HA state - as noted in the commit. I would also include it...

**Comment by ytti** [comment]
Hmm this seems like it's already fixed in c59b4b5c
